config-overrides.js

```diff
@@ -1,6 +1,7 @@
 const webpack = require('webpack');
 module.exports = function override(config) {
     const fallback = config.resolve.fallback || {};
+    const NodePolyfillPlugin = require("node-polyfill-webpack-plugin")
```

Comment:
why you need this plugin?

Reply:
It was originally to fix an issue with one of the test we were writing, but it became unnecessary. Been removed!